NeuroID/src/main/java/com/neuroid/tracker/NeuroID.kt

```diff
@@ -173,8 +185,10 @@ class NeuroID
             }
         }
 
-        data class Builder(var application: Application? = null, var clientKey: String = "") {
-            fun build() = NeuroID(application, clientKey)
+        data class Builder(val application: Application? = null,
+                           val clientKey: String = "",
+                           val serverEnvironment: String= "prod") {
```

Comment:
everywhere else we have "production" and not "prod? should we change it here?

Reply:
yeah this should be "production" will update this.